test/flowline_test.py

```diff
@@ -0,0 +1,142 @@
+# Copyright (C) 2019-2020 by Daniel Shapero <shapero@uw.edu>
```

Comment:
We can make both of these tests into special cases of the existing counterparts by parameterizing those tests. I can show you how to do that or if you want an example you could look at the inverse test, which is parameterized over the solver.